settings/custom-styling.mdx

```diff
@@ -1,10 +1,7 @@
 ---
 title: "Custom Styling"
-description: "Customize your Mintlify site in mint.json"
```

Comment:
Let's try to use as many descriptions as possible. It's better for SEO and styling